support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx

```diff
@@ -88,23 +89,45 @@ export type BenefitsCheckListProps = {
 	cssOverrides?: SerializedStyles;
 };
 
+type SvgBulletProps = {
+	opacity?: number;
+};
+function SvgBullet({ opacity = 1 }: SvgBulletProps): JSX.Element {
```

Comment:
I know this is a small svg but I think we should move svg markup to their own modules/files, it's code we're unlikely to edit so could be moved out to make this module easier to read

Reply:
Ok have moved it out to own file within same folder (as only used here). Think thats the last change but have had to  do more copy/css styling tweaks to keep up with new updates in the Figma file (probably from user feedback)